Examples/Python/setup.sh.in

```diff
@@ -1,3 +1,28 @@
-python_dir=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}"  )" &> /dev/null && pwd  )
+# This file is part of the Acts project.
+#
+# Copyright (C) 2016-2020 CERN for the benefit of the Acts project
```

Comment:
If you put the statement, you could update the year.

Reply:
Good point :thinking: